eZ/Publish/API/Repository/Tests/Values/User/Limitation/LanguageLimitationTest.php

```diff
@@ -490,6 +492,30 @@ public function testPublishVersionTranslationWhenUserHasNoAccessToAllLanguages()
         $contentService->publishVersion($draft->versionInfo, [self::GER_DE, self::ENG_GB]);
     }
 
+    /**
+     * @throws \eZ\Publish\API\Repository\Exceptions\BadStateException
+     * @throws \eZ\Publish\API\Repository\Exceptions\ForbiddenException
+     * @throws \eZ\Publish\API\Repository\Exceptions\NotFoundException
+     */
+    public function testCopyContentWithLanguageLimitationDifferentThanInitialLanguageCode(): void
+    {
+        $repository = $this->getRepository();
+        $contentService = $repository->getContentService();
+        $permissionResolver = $repository->getPermissionResolver();
+
+        $draft = $this->createMultilingualFolderDraft($contentService);
+        $content = $contentService->publishVersion($draft->getVersionInfo());
+
+        $permissionResolver->setCurrentUserReference(
+            $this->createEditorUserWithLanguageLimitation([self::GER_DE])
+        );
+
+        $locationCreateStruct = new LocationCreateStruct(['parentLocationId' => 2]);
+        $content = $contentService->copyContent($content->contentInfo, $locationCreateStruct);
+
+        self::assertInstanceOf(Content::class, $content);
```

Comment:
This assert will never throw because `copyContent` has strict `: Content` return type (it will result in a fatal error on a `copyContent` call). Maybe some Content load by ID with assertion of existing translations in a new subtree?